util/buildbot/common.sh

```diff
@@ -51,10 +51,10 @@ get_sources () {
 # sets $runtime_dlls
 find_runtime_dlls () {
 	local triple=$1
-	# Try to find runtime DLLs in various paths
+	# Try to find runtime DLLs in various paths (fun)
 	local tmp=$(dirname "$(command -v $compiler)")/..
 	runtime_dlls=
-	for name in lib{clang_rt,c++,unwind,winpthread-}'*'.dll; do
+	for name in lib{c++,unwind,winpthread-}'*'.dll; do
```

Comment:
I don't think winpthread is needed either, since Luanti (actually JThread or wherever that code comes from) uses native Win32 threading and I remove it for my Windows builds without an issue

Reply:
seems to be true (maybe since we switched to clang?) but I find it harder to guarantee that none of the other libraries are depending on pthread.

Reply:
Fair enough. There's no explicit linkage from the dependencies we have from back when I checked the linkage tree, but don't know if there is some dynamic loading at runtime that I might have missed (though I would probably have still noticed that...).